includes/AddOns/SteelDesign/UltimateConfigurationEC3.js

```diff
@@ -0,0 +1,330 @@
+/*
```

Comment:
It is for that reason, because:
Fire resistance configuration is only for EN standard and NTC one (for this isn't no API support)
Serviceability configuration is small and has functions for individual standards